### PR TITLE
improvement(ami/gce artifact test): Scylla configuration that was asked in user-data is read and set

### PR DESCRIPTION
**Pull request by juliayakovlev**
Make sure that in the GCE/AMI artifacts test Scylla configuration that was asked in
user-data is read and set.
Check that cluster name is set.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: artifacts_test.py [hunk 1]
```diff
@@ -27,6 +28,14 @@ def node(self):
             raise ValueError('DB cluster has not been initiated')
         return self.db_cluster.nodes[0]
 
+    def check_cluster_name(self):
+        with self.node.remote_scylla_yaml(SCYLLA_YAML_PATH) as scylla_yml:
+            yaml_cluster_name = scylla_yml.get('cluster_name') or ''
```
Comment (bentsi) [inline artifacts_test.py:33]: why are you using or?
```python
scylla_yml.get("cluster_name", "")
```

File: artifacts_test.py [hunk 2]
```diff
@@ -27,6 +28,14 @@ def node(self):
             raise ValueError('DB cluster has not been initiated')
         return self.db_cluster.nodes[0]
 
+    def check_cluster_name(self):
+        with self.node.remote_scylla_yaml(SCYLLA_YAML_PATH) as scylla_yml:
```
Comment (bentsi) [inline artifacts_test.py:32]: better `scylla_yaml`

**Review by bentsi — COMMENTED** [review]
Looks ok, minor comments

File: artifacts_test.py (hunk 1 above)
Comment (juliayakovlev) [reply artifacts_test.py]: changed

File: artifacts_test.py (hunk 2 above)
Comment (juliayakovlev) [reply artifacts_test.py]: typo

**Review by bentsi — APPROVED** [review]

**Pull request merged**